wasmer/wat.go

```diff
@@ -0,0 +1,38 @@
+package wasmer
+
+// #include <wasmer_wasm.h>
+//
+// wasm_byte_vec_t *to_wat2wasm(uint8_t *wat_data, size_t wat_length) {
+//     wasm_byte_vec_t wat;
+//     wat.size = wat_length;
+//     wat.data = (wasm_byte_t*) wat_data;
+//
+//     return wat2wasm(&wat);
+// }
+import "C"
+import (
+	"runtime"
+	"unsafe"
+)
+
+func Wat2Wasm(wat string) ([]byte, error) {
```

Comment:
Why not using a byte array directly here? I think it may ease some call and also, avoid converting the byte array to a string in `module.go`